### DISCUSSION TITLE
<unknown> as a Sentry issue title for JS errors

### DISCUSSION THREAD

**Discussion by ehausen** [stayallive/wp-sentry#115] [Q&A] [answered]
Using WP plugin version 4.7.0

When a new Javascript issue is created the Sentry issue gets a title: `<unknown>`

PHP issues from the same site seem to get a title just fine.

![CleanShot 2021-11-08 at 15 00 12](https://user-images.githubusercontent.com/6204129/140746279-c7f6b8c0-0626-4caa-9d0a-0f96af0ee617.png)

**Comment by stayallive** [comment, marked as answer]
Hmm, as far as I know we don't do anything special that causes this. Capturing JavaScript errors is pretty wonky though. But just in case this was a JS SDK problem I've updated it to the latest release (plugin version 4.8.0) so please update and see if the problem persists.

Also try the JavaScript tester under the tools section in your WP admin to see if that event comes through fine. If so, the issue is most likely the Sentry SDK was unable to resolve a nice message and substituted unknown, it should not happen for every event though.
